app/controllers/pets_applications_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PetsApplicationsController < ApplicationController
+  def new
+  end
+
+  def show
+    @searched_pets = pet_search
+    @application = PetsApplication.find(params[:id])
+    @applicant = Applicant.retrieve_applicant(params[:id])
+    @pets = @applicant.pets
+    @status = PetsApplication.check_app_status(@applicant)
+  end
+
+  def create
+    applicant = Applicant.new(applicant_params)
+    if applicant.save
+      application = PetsApplication.create!(applicant: applicant)
```

Comment:
In here I think you could create a new association by passing `PetApplication.create!` both a `pet_id` and an `application_id`